### ISSUE TITLE
Data not available

### ISSUE THREAD

**Issue by timlinux** [kartoza/clean-cooking-platform#332]
![image](https://user-images.githubusercontent.com/178003/154284253-5cc76712-e91b-44b3-874d-399ee2257efb.png)
![image](https://user-images.githubusercontent.com/178003/154284326-05af682c-2836-4edb-b64e-aa73530222a7.png)

Also cannot query layer by clicking it

**Comment by babakkhavari** [comment]
Yes, something happens with all the polygons once they clipped. The styling changes and there is no data available. 
![image](https://user-images.githubusercontent.com/35455877/155289327-41821d8c-9dff-401d-898c-ffa148b19071.png)
